main/lsp/LSPPreprocessor.cc

```diff
@@ -36,25 +36,12 @@ void cancelTimer(unique_ptr<Timer> &timer) {
     }
 }
 
-class QueueOutput final : public LSPOutput {
```

Comment:
I previously introduced this class to enable LSPPreprocessor to send ShowOperation notifications to the client by delegating it to the main message processing thread.

Now, LSPPreprocessor can send messages to the client *directly* via a shared LSPOutput object.